eng/Versions.props

```diff
@@ -42,6 +42,7 @@
     <dotnetdevcertsPackageVersion>3.1.1-servicing.19615.10</dotnetdevcertsPackageVersion>
     <dotnetusersecretsPackageVersion>3.1.1-servicing.19615.10</dotnetusersecretsPackageVersion>
     <dotnetwatchPackageVersion>3.1.1-servicing.19615.10</dotnetwatchPackageVersion>
+    <MicrosoftAspNetCoreBlazorTemplatesPackageVersion>3.2.0-preview2.20110.5</MicrosoftAspNetCoreBlazorTemplatesPackageVersion>
```

Comment:
Are we going to be able to ingest this via maestro?

Reply:
Does this mean have it push updates via darc \ maestro? Let me consult the build team. At this point, we were planning on manually updating it once we were happy with a specific builds.

Reply:
It worries me if there is anything manual being added for this.

Reply:
I've updated the Version.Details.xml file to include a ref to the Blazor template. We should be able to set up a darc subscription once the 3.1.3xx branch is available.